symengine/lib/symengine_wrapper.pyx

```diff
@@ -1550,6 +1688,18 @@ ACsch = acsch
 ATan2 = atan2
 Log = log
 Gamma = gamma
+LambertW = lambertw
```

Comment:
Aliases above were for backwards compatibility. Newer functions do not need this